app/src/main/java/io/homeassistant/companion/android/webview/WebViewPresenterImpl.kt

```diff
@@ -179,6 +161,36 @@ class WebViewPresenterImpl @Inject constructor(
         }
     }
 
+    override fun setStatusbarColor(colorString: String) {
+        mainScope.launch {
+            try {
+                val statusbarNavBarColor = parseColorWithRgb(colorString)
+                view.setStatusBarAndNavigationBarColor(statusbarNavBarColor)
+                return@launch
+            } catch (e: Exception) {
+                Log.e(TAG, "Issue getting/setting theme from webview. Get/Set theme from HA", e)
```

Comment:
It will be nice to see timber logs https://github.com/JakeWharton/timber as it is very popular

Reply:
I think that would be a extra PR to change the logging framework

Reply:
Yea out of scope we should get a feature request opened so we can start a discussion there on what the requirements are.